winml/lib/Api.Image/D3DDeviceCache.cpp

```diff
@@ -49,37 +49,48 @@ D3DDeviceCache::D3DDeviceCache(winml::LearningModelDeviceKind const& deviceKind)
     return;
   }
 
-  DXGI_GPU_PREFERENCE preference;
-  WINML_THROW_IF_FAILED(GetGPUPreference(deviceKind, &preference));
-
   CommonDeviceHelpers::AdapterEnumerationSupport support;
   WINML_THROW_IF_FAILED(CommonDeviceHelpers::GetAdapterEnumerationSupport(&support));
 
   const char noHardwareAdaptersAvailableErrStr[] = "No hardware adapters available";
   const char failedToObtainHardwareAdaptersErrStr[] = "Failed to obtain hardware adapters.";
   HRESULT hardwareAdapterSuccessfullyObtained = S_OK;
-  if (support.has_dxgi) {
-    winrt::com_ptr<IDXGIAdapter1> spAdapter;
-    hardwareAdapterSuccessfullyObtained = GetDXGIHardwareAdapterWithPreference(preference, spAdapter.put());
+
+  if (support.has_dxcore && deviceKind == winml::LearningModelDeviceKind::ComputeAccelerator) {
+    winrt::com_ptr<IDXCoreAdapter> spAdapter;
+    hardwareAdapterSuccessfullyObtained = GetDXCoreHardwareAdapterForVPU(spAdapter.put());
     if (hardwareAdapterSuccessfullyObtained == HRESULT_FROM_WIN32(ERROR_NOT_FOUND)) {
       WINML_THROW_HR_MSG_NO_TELEMETRY_SENT(hardwareAdapterSuccessfullyObtained, noHardwareAdaptersAvailableErrStr);
     } else {
       WINML_THROW_IF_FAILED_MSG(hardwareAdapterSuccessfullyObtained, failedToObtainHardwareAdaptersErrStr);
     }
     WINML_THROW_IF_FAILED(D3D12CreateDevice(spAdapter.get(), D3D_FEATURE_LEVEL_11_0, IID_PPV_ARGS(device_.put())));
```

Comment:
>D3D_FEATURE_LEVEL_11_0 [](http://example.com/codeflow?start=61&length=22)

For MCDM devices, this needs to be D3D_FEATURE_LEVEL_1_0_CORE since FL_11_0 isn't supported on non-GPUs